app/models/course/discussion/topic.rb

```diff
@@ -7,6 +7,10 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  def to_partial_path
+    'course/discussion/topic'.freeze
```

Comment:
there's 2.3 for the ppa we're using. I'll upgrade soon.
https://www.brightbox.com/docs/ruby/ubuntu/